client-app/router/index.ts

```diff
@@ -22,11 +24,10 @@ export function createRouter(options: { base: string }) {
 
   router.beforeEach((to, _from, next) => {
     // Protecting routes
-    if (
-      (to.meta.requiresAuth || !storeSettings.anonymousAccessEnabled) &&
-      !isAuthenticated.value &&
-      to.name !== "SignIn"
-    ) {
+    const unauthorizedAccessIsDenied: boolean =
```

Comment:
It is not necessary to write the type here. In my case Lint shows an error 'Type boolean trivially inferred from a boolean literal, remove type annotation.'